### ISSUE TITLE
Doesn’t seem to acknowledge projects

### ISSUE THREAD

**Issue by ethereal-engineer** [bradyt/taskw-dart#10]
The app doesn't seem to acknowledge projects nor any task that is assigned to a project 🤷🏻

**Comment by bradyt** [comment]
Can I interpret this as you'd like to see and edit a task's project fields, and to sort and filter the task list by project?

By the way, my apprehension was due to [Deprecation of project in favor of tags](https://github.com/GothenburgBitFactory/taskwarrior/issues?q=Deprecation+of+project+in+favor+of+tags). But I see that issue was closed two weeks ago.

**Comment by bradyt** [comment]
Here is the changelog for `v0.1.2`.

```
- Add projects feature
  + Project is displayed in list and detail views
  + Simple UI to edit project field
  + Sort by project
  + Very basic project filter
- Add search feature
```

Implementation is at https://github.com/bradyt/taskw-dart/commit/b1054819c94d3b389520256937af3910b67548cb.

It's in review for App Store and Play Store. I expect it to be on App Store in less than a day, and available for Android users in a few days.

I can ping thread when it's on App Store.

I will close now, as the feature as described is implemented.

Ideas for further improvements to project feature are welcome, maybe as a new issue, for easier tracking. In parallel, I will be considering how I might improve the project editing or filtering UIs, but it seemed sufficient for getting to release and reassessing priorities.

Small point: TaskwarriorC2 abbreviated the projects in list view. I haven't considered what the calculation is for that, nor if they implemented that or inherited it from the compiled Taskwarrior.

**Comment by bradyt** [comment]
Version `0.1.2` is now available on App Store.

**Comment by ethereal-engineer** [comment]
Wow. Fantastic work! I haven't checked it out yet. It seems that despite resolving my Mail problem GitHub is still not sending notifications to me.

I guess I assumed that projects would be covered in core basics because they are exemplified in task warrior's best practices notes. 

I didn't know about even the idea to ditch projects in favour of tags, and it seems kind of silly to me. A project represents the single aim and end goal - the tangible "project over" part, and keeps all your tasks accountable to purpose. Tags are myriad, and can represent contexts, and categorisation. As a decades long user of OmniFocus, I use both!

Anyway, thank you for addressing this. I will check out the app now.
